src/type/scalars.js

```diff
@@ -87,7 +97,7 @@ export const GraphQLFloat = new GraphQLScalarType({
   },
 });
 
-function coerceString(value: mixed): ?string {
+function coerceString(value: mixed): string {
```

Comment:
Oh interesting: technically the spec lets us coerce arrays into strings and booleans: https://github.com/facebook/graphql/blob/master/spec/Section%203%20--%20Type%20System.md#string

However it does NOT allow us to coerce arrays into floats or ints.